api/src/main/java/org/pmiops/workbench/actionaudit/auditors/SumoLogicAuditor.kt

```diff
@@ -0,0 +1,18 @@
+package org.pmiops.workbench.actionaudit.auditors
+
+import org.pmiops.workbench.model.EgressEvent
+import org.pmiops.workbench.model.EgressEventRequest
+
+interface SumoLogicAuditor {
```

Comment:
FWIW, this interface and implementation don't need to be in Kotlin. Java classes can implement Kotlin interfaces and vice-versa. See `UserServiceAuditor.java`.

Reply:
Woops – let's keep this between you and me :)

I took it as a chance to try and get a bit more familiar with Kotlin before we need to make a team decision on whether or not to adopt it more widely. That's good to know it's not as infectious as I might have worried.